src/main/resources/application.yml

```diff
@@ -34,6 +34,8 @@ security:
 
 identity:
   url: ${OAUTH_URL:http://localhost:8080}
+  signoutUrl: ${OAUTH_URL:http://localhost:8080}/logout
+  returnToUrl: ${IDM_URL:http://localhost:8081}/mgmt/login
```

Comment:
returnToUrl can be renamed to idmLoginUrl to make it login name in sync with its value.